Jenkinsfile

```diff
@@ -128,9 +122,26 @@ ansiColor('xterm') {
           }
 
           if (currentBuild.result == 'SUCCESS'){
+            stage('Bump version'){
+              sh '''#!/bin/bash -ex
+              source ~/.nvm/nvm.sh
```

Comment:
At some point, we should see about either (a) replacing the executor dockerfile with something that doesn't require us to call nvm in every shell script or (b) do something like what the sdk and do all the real work in a separate container (with the way we have docker setup, there should be minimal overhead - the sdk's overhead is from running multiple containers in parallel on the same box).